tests/FINDOLOGIC/Api/Tests/RequestBuilders/Json/SuggestionRequestBuilderTest.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace FINDOLOGIC\Api\Tests\RequestBuilders\Json;
+
+use FINDOLOGIC\Api\Config;
+use FINDOLOGIC\Api\Exceptions\InvalidParamException;
+use FINDOLOGIC\Api\Exceptions\ParamNotSetException;
+use FINDOLOGIC\Api\RequestBuilders\Json\SuggestionRequestBuilder;
+use FINDOLOGIC\Api\Tests\TestBase;
+
+class SuggestionRequestBuilderTest extends TestBase
```

Comment:
Here and in other places, preferably use `Suggest` instead of `Suggestion` for consistency with the rest of our stuff.